sql/src/main/java/io/crate/analyze/relations/SubselectRewriter.java

```diff
@@ -156,15 +154,26 @@ public AnalyzedRelation visitMultiSourceSelect(MultiSourceSelect multiSourceSele
      *         It tries to preserve the alias/name of the parent if possible
      */
     private static Collection<Path> namesFromOutputs(Collection<? extends Symbol> outputs,
-                                                     Map<Symbol, Field> replacedFieldsByNewOutput) {
+                                                     Map<Symbol, Field> replacedFieldsByNewOutput,
+                                                     QueriedSelectRelation parent) {
         List<Path> outputNames = new ArrayList<>(outputs.size());
         for (Symbol output : outputs) {
             Field field = replacedFieldsByNewOutput.get(output);
             if (field == null) {
                 if (output instanceof Path) {
                     outputNames.add((Path) output);
                 } else {
-                    outputNames.add(new OutputName(SymbolPrinter.INSTANCE.printSimple(output)));
+                    // There might be an alias/output-name on the parent relation which does not occur in the parent query spec
+                    // Th
+                    // We only support 1 column and only single-row subselects
```

Comment:
This means as soon as more columns are selected it breaks again? I think we'll need a different approach